src/librustc_codegen_ssa/back/link.rs

```diff
@@ -140,7 +140,7 @@ pub fn link_binary<'a, B: ArchiveBuilder<'a>>(
 // The third parameter is for env vars, used on windows to set up the
 // path for MSVC to find its DLLs, and gcc to find its bundled
 // toolchain
-fn get_linker(sess: &Session, linker: &Path, flavor: LinkerFlavor) -> Command {
+fn get_linker(sess: &Session, linker: &Path, flavor: LinkerFlavor, fallback: bool) -> Command {
```

Comment:
`fallback` -> `self_contained` everywhere, since https://github.com/rust-lang/rust/pull/72999 fixed the terminology.

Reply:
Replaced all `fallback` variables , there are still various `*_fallback` functions but that seems out of scope for this PR.